src/pages/gen2/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -289,20 +289,53 @@ If you are using the [Authenticator component](https://ui.docs.amplify.aws/react
 Enabling a signing in of a user with a social provider can be done as shown in the following code snippet.
 
 ```ts title="src/my-client-side-js.js"
-import { signInWithRedirect } from 'aws-amplify/auth';
+import { signInWithRedirect, getCurrentUser, fetchUserAttributes } from 'aws-amplify/auth';
```

Comment:
Now we are not using these two APIs here we can remove them and add to the next example.